lib/_stylesheets/custom/_markdown.scss

```diff
@@ -0,0 +1,19 @@
+// markdown-it-container with `card` expression
```

Comment:
Should card styling not be separated out like the linked file? 

Although, it still makes sense having markdown specific css here.
 
See: https://github.com/nhsbsa/nhsbsa-digital-playbook/pull/152/files#diff-4c74c34129cd3ce69305329a17c581e95e6d1cac7f48a2bb8ea133841beba9b0

Reply:
I think the nunjucks card component should be treated separatedly from the styling of markdown.
We could extract into multiple `md_*` files in future if MD styling grows to much.